h/accounts/schemas.py

```diff
@@ -191,16 +191,16 @@ class RegisterSchema(CSRFSchema):
             unique_username,
             unblacklisted_username,
         ),
-        title=_('Username:'),
+        title=_('Username'),
         hint=_('between {min} and {max} characters, containing only letters, '
                'numbers, periods, and underscores').format(
```

Comment:
When we remove the legacy forms I guess we should remove these hints, they aren't used anymore